### PR TITLE
add short README for each variant

### PR DESCRIPTION
**Pull request by tobsan**
Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: conf/variant/intel/README [hunk 1]
```diff
@@ -0,0 +1 @@
+Variant configuration for building PELUX for the Intel NUC without Qt+QtAuto
```
Comment (DunderRoffe) [inline conf/variant/intel/README]: Write headless or something instead of without Qt+QtAuto. Otherwise it wont scale when adding additional variants

Comment (breakreturn) [reply conf/variant/intel/README]: Non-graphical is used to distinguish this in a project I'm in. "Headless" means very different things to people it turns out :)

**Comment by tobsan** [comment]
@DunderRoffe @breakreturn updated now.

File: conf/variant/rpi/README [hunk 2]
```diff
@@ -0,0 +1 @@
+Variant configuration for building a non-graphical PELUX for the Raspberry Pi 3.
```
Comment (GordanM) [inline conf/variant/rpi/README]: For consistency reasons all README files should end with a ".".

**Pull request merged**